app/Model/ObjectTemplate.php

```diff
@@ -316,4 +316,41 @@ public function setActive($id)
         }
         return 1;
     }
+
+    public function getRawFromDisk($uuidOrName)
```

Comment:
does this function need to be public, or could it be private like readTemplatesFromDisk ?

Reply:
This function is used by the controller and could be used for other purpose.